test/integration/api_query_test.lua

```diff
@@ -415,6 +418,7 @@ function g.test_servers()
                 membership = {status = 'alive'},
                 vshard_router = box.NULL,
                 vshard_storage = {buckets_active = 3000},
+                general = { http_port = 8084 },
```

Comment:
API looks a bit incomplete. Because currently we return only local http port. Fair solution requires map-reduce over all instances. And it's a reason why it wasn't done before.
However AFAIK it's a possible to get set of requested fields inside graphql callback (using the first argument). So, we can perform map-reduce only if user explicitly request http_port.